pkg/injector/allowedsawatcher/predutil.go

```diff
@@ -0,0 +1,108 @@
+package allowedsawatcher
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/dapr/dapr/utils"
+)
+
+type equalPrefixLists struct {
+	equal  []string
+	prefix []string
+}
+
+const minPrefixLength = 4
```

Comment:
at one moment was to avoid possibly hitting system namespaces but that was added already. I guess there is no need, as it could also be that a user wants all SAs matching a prefix on any namespace (`*:sa*`), os all SAs matching with a namespace matching a prefix (`ns*:*`). I'll take care. thanks